test/Riok.Mapperly.Tests/_snapshots/UserMethodTest.StaticMapperShouldEmitDiagnosticForNonStaticMethods.00.verified.txt

```diff
@@ -0,0 +1,17 @@
+﻿{
+  Diagnostics: [
+    {
+      Id: RMG001,
+      Title: Has an unsupported mapping method signature,
+      Severity: Error,
+      WarningLevel: 0,
+      Location: : (10,4)-(10,54),
+      Description: ,
+      HelpLink: ,
+      MessageFormat: {0} has an unsupported mapping method signature,
+      Message: NonStaticToObject has an unsupported mapping method signature,
```

Comment:
Would it be possible to emit a custom error message for this case? Right now, it is rather generic

Reply:
Improved.